trailblazer/store/filters/analyses_filters.py

```diff
@@ -80,6 +80,10 @@ def filter_analyses_by_empty_comment(analyses: Query, **kwargs) -> Query:
     return analyses.filter(sqlalchemy.or_(Analysis.comment.is_(None), Analysis.comment == ""))
 
 
+def filter_analyses_by_most_recent(analyses: Query, **kwargs) -> Query:
+    return analyses.order_by(Analysis.started_at.desc()).limit(1)
```

Comment:
Feels slightly odd to call analyses and only want a single analysis back. Maybe this should be a different endpoint for cases? Might be too much of a rewrite though.

Reply:
And would it ever be set in isolation?

Reply:
True, it was not really necessary - the default sort field and order is enough. Removed it